machida/lib/wallaroo/experimental/connector_wire_messages.py

```diff
@@ -9,15 +9,19 @@
 
 
 class Hello(object):
```

Comment:
I'm going to make some kind of assumption that this is all correct but it might make sense to run a short script against a file and round-trip it as a series of frames in a byte string for an easy sanity check. We can use this file against both parsers which should at least ensure most things are working (bitflag order and other details are more likely caught with running apps and in the longer run we should have fuzzers on all of our protocols to ensure we gracefully fail rather than load in 4GB of data if someone flips some overly significant set of bits to all 1s, cosmic rays of course 😉).